internal/service.go

```diff
@@ -89,6 +91,21 @@ func moduleEnabledInEnv(identifier string) bool {
 	return false
 }
 
+// moduleEnabledInProductFiles returns true if the provided `identifier` is
+// a name of a file in the  /etc/product.d/*.prod, otherwise false.
+func moduleEnabledInProductFiles(identifier string) bool {
+	files, err := ioutil.ReadDir("/etc/products.d")
+	if err != nil {
+		return false
+	}
+	for _, file := range files {
+		if identifier == strings.TrimSuffix(file.Name(), ".prod") {
```

Comment:
Is `.prod` the only extension which can happen there? If so, then I'm fine with that change. :)

If not, then we may remove the extension via:
https://golang.org/pkg/path/filepath/#Ext